games/Hangman.py

```diff
@@ -0,0 +1,128 @@
+import random
+
+# Simple terminal hangman game with ASCII art
+def choose_random_word():
```

Comment:
Trivial: This function is missing it's docstring 🙂

Reply:
Sorry I missed that, its added now!